### PR TITLE
Upgrade winston to v3

### PR DESCRIPTION
**Pull request by KevinGrandon**
* Update flow-typed library.
* Upgrade call signature to match v3 createLogger interface.
* Remove trace and fatal methods as they are not winston log levels.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,50 +112,44 @@ A [Winston](https://github.com/winstonjs/winston) configuration object. Optional
...
-logger.trace(...args)
+logger.log(level, ...args);
 ```
 
+* `level: string` - Valid levels: `'error'`,`'warn'`,`'info'`,`'verbose'`,`'debug'`,`'silly'`
 * `args: [string]`
 
 ```js
-logger.debug(...args)
+logger.error(...args);
```
Comment (KevinGrandon) [inline README.md]: Now just sorted in the order of flow-types for ease of maintenance.

File: src/__tests__/index.node.js [hunk 2]
```diff
@@ -58,17 +59,26 @@ test('Server logger', async t => {
...
+    }: {
+      level: SupportedLevelsType,
+      message: string,
+      hello: string,
+    }) {
       t.equals(level, 'info', 'level is ok');
       t.equals(message, 'test', 'message is ok');
       t.equals(message, 'test', 'message is ok');
-      t.deepLooseEqual(meta, {hello: 'world'}, 'meta is ok');
+      t.equals(hello, 'world', 'meta is ok');
```
Comment (KevinGrandon) [inline src/__tests__/index.node.js]: winston v3 no longer maintains a separate meta field, and instead merges everything.

File: README.md [hunk 3]
```diff
@@ -40,33 +40,33 @@ app.middleware({logger: LoggerToken}, ({logger}) => {
...
-      transports: [
-        new winston.transports.File({filename: 'logs.log'}),
-      ],
+      transports: [new winston.transports.File({filename: 'logs.log'})],
     };
     app.register(UniversalLoggerConfigToken, config);
   }
   return app;
-}
+};
```
Comment (AlexMSmithCA) [inline README.md]: https://mislav.net/2010/05/semicolons/

**Comment by AlexMSmithCA** [comment]
Awesome!  Thanks for handling this.

File: README.md (hunk 3 above)
Comment (KevinGrandon) [reply README.md]: I agree, remove all semicolons! (This was prettier though...)

**Pull request merged**